### PR TITLE
Created `$client` Functions

### PR DESCRIPTION
**Pull request by Neodevils**
## Added Function

* **Client Avatar**, _`$clientAvatar[size?;format?;dynamic?]`_
* **Client ID**, _`$clientId`_
* **Client Token**, _`$clientToken`_

### CODE REVIEW

File: src/functions/discord/client/$clientAvatar.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    description: "Returns the Avatar of client",
+    code: (data: funcData, scope: Scope[]) => {
+        // Adding default values
+        const [size = 4096, dynamic = "true", format = "png"] = data.splits;
+
+        // Getting the current scope
+        const currentScope = scope[scope.length - 1];
+
+        // Getting the client avatar
+        const clientAvatar = `__$DISCORD_DATA$__.client.AvatarURL({format: ${format}, dynamic: ${dynamic}, size: ${size}})`;
```
Comment (USERSATOSHI) [inline src/functions/discord/client/$clientAvatar.ts]: this should be 
```js
`__$DISCORD_DATA$__.client.readyData.user.avatarUrl({ format: "${format}" , dynamic: ${dynamic}, size: ${size} })`
```

File: src/functions/discord/client/$clientAvatar.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+        // Returning the result
+        const res = escapeResult(`(${clientAvatar})`);
```
Comment (USERSATOSHI) [inline src/functions/discord/client/$clientAvatar.ts]: escapeResult(clientAvatar) is fine enough here

**Review by USERSATOSHI — CHANGES_REQUESTED** [review]

**Comment by USERSATOSHI** [comment]
Also seems like you didn't add $clientAvatar.ts to index.ts of client folder
also update that here

**Review by USERSATOSHI — APPROVED** [review]

**Comment by Neodevils** [comment]
Renamed the branch #448

**Pull request closed without merging**